bin/aapp_dr_runner.py

```diff
@@ -262,7 +253,7 @@ def move_aapp_log_files(config):
                 base_filename = os.path.basename(file_name)
                 dst = os.path.join(destination, base_filename)
                 shutil.move(file_name, dst)
-            except Exception as err:
+            except Exception:
                 LOG.warning(
                     "Failed to move log file: {} to: {}".format(file_name, dst))
```

Comment:
Perhaps consider `LOG.exception` here or otherwise adding some information extracted from `err` on why it failed?

Reply:
Ok, added